var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -109,6 +115,7 @@ class Dealii(CMakePackage):
 
     # optional dependencies
     depends_on("mpi",              when="+mpi")
+    depends_on("adol-c@2.6.4:",    when='@9.0:+adol-c')
```

Comment:
They have not yet made a point release that fixes a really minor issue that affects print functions / our test suite. We either need to require `adol-c@dev` for now or we could just apply [this little patch](https://gitlab.com/adol-c/adol-c/commit/9af81273bfa01ea37e6a4e1893dd5e07fea6e467) to `adol-c@2.6.3`.